packages/extension-utils/src/OAuthConfigForm/OAuthConfigForm.tsx

```diff
@@ -312,33 +317,35 @@ export const OAuthConfigForm = ({
               />
             </Fieldset>
           </Form>
-          {!!fields.webUrl && (
-            <>
-              <Paragraph fontSize="small">
-                On {fields.webUrl}, enable {clientLabel} as a{' '}
-                <Link
-                  href="https://github.com/looker-open-source/sdk-codegen/blob/main/docs/cors.md#reference-implementation"
-                  target="_blank"
-                >
-                  Looker OAuth client
-                </Link>{' '}
-                by adding "{(window as any).location.origin}" to the{' '}
-                <Link href={`${fields.webUrl}/admin/embed`} target="_blank">
-                  Embedded Domain Allowlist
-                </Link>
-                . If API Explorer is also installed, the configuration below can
-                be used to{' '}
-                <Link
-                  href={`${fields.webUrl}/extensions/marketplace_extension_api_explorer::api-explorer/4.0/methods/Auth/register_oauth_client_app`}
-                  target="_blank"
-                >
-                  register this API Explorer instance
-                </Link>{' '}
-                as an OAuth client.
-              </Paragraph>
-              <CodeCopy key="appConfig" language="json" code={appConfig} />
-            </>
-          )}
+          {fields.fetchedUrl &&
+            fields.fetchIntent === CRITICAL &&
+            fields.fetchResult !== '' && (
```

Comment:
We shouldn't be making these configuration steps harder to see. Showing the configuration setup is a feature, not a bug, even when a specific user might have a successful oauth config verification. We don't have this information clearly documented anywhere else in all of Looker.

Reply:
These steps are showing only when the verification / save is successful. Now it will always show if there are any issues. You are saying you would prefer the help message to update as the user types into the URL box? Even if URL is invalid?